nodes.py

```diff
@@ -5,6 +5,11 @@
 import torch.nn.functional as F
 from diffusers import ConsistencyDecoderVAE
 from torch import Tensor
+import comfy.sd
+import comfy.utils
+from huggingface_hub import hf_hub_download
+
+os.environ["HF_HUB_ENABLE_HF_TRANSFER"] = "1"
```

Comment:
I'm not sure this works after importing hf_hub_download, because by then, hf has already checked if the flag exists.
It would work on the modal setup because we set it to "1" during the build process.

Reply:
You are right. It is set as a constant as soon as it is loaded. If another plugin loads it first, we won't be able to set it as all in this plugin. We will have to do it as an runtime env.